README.md

```diff
@@ -1,26 +1,122 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+while loop-
+
+let num = 20
+while (num > 0) {
+  if (num % 2) {
+    console.log(num);
+  }
+  num--
+}
+
+
+for loop-
+
+for (let num = 20; num > 0; num--) {
+  if (num % 2) {
+    console.log(num);
+  }
+}
+
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+for loop-
+
+for (let num = 0; num <= 10; num++){
+  if (num % 2 === 0){
+    console.log(num + ' is even')
+  } else {
+    console.log(num + ' is odd')
+  }
+}
+
+
+while loop-
+
+let num = 0
+while (num < 10) {
+  if (num % 2) {
+    console.log(num + ' is even');
+  } else {
+    console.log(num + ' is odd');
+  }
+  num++
+}
+
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
 For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
+while loop-
+
+let num = 0
+while (num <= 20) {
+  if (num % 2) {
```

Comment:
There is No need to do (num % 2) here. This returns a remainder (which would help us check for even and odds). Also this was supposed to go from 0 to the num.